lib/trmnl/i18n/locales/web_ui/de-AT.yml

```
@@ -454,21 +454,21 @@ de-AT:
       plugin_uuid_hint: Verwende diese Kennung für Anfragen an die Webhook-API.
       preview: Vorschau – dein Gerät wird aktualisierte Daten anzeigen
       recipe_cta_heading_published: Diese Erweiterung ist eine Blaupause
-      recipe_cta_heading_unlisted: This plugin is unlisted
+      recipe_cta_heading_unlisted: Diese Erweiterung ist nicht gelistet
       recipe_cta_heading_in_review: Blaupause wird geprüft
       recipe_cta_heading_not_published: Als Blaupause veröffentlichen?
       recipe_cta_body_published: Änderungen werden sich auf aktuelle und zukünftige Installationen/Forks auswirken.
       recipe_cta_body_in_review: Diese Erweiterung wurde an unser Team übermittelt. Danke für deine Geduld!
       recipe_cta_body_not_published: Reiche diese Erweiterung ein, damit sie auch andere installieren können. __Mehr erfahren__.
       recipe_cta_confirm: Bist du sicher? Unser Team wird sich in einigen Tagen nach erfolgter Prüfung zurückmelden. Du kannst weiterhin Änderungen vornehmen.
-      recipe_visibility_title: How would you like to publish?
-      recipe_visibility_edit_title: Edit Submission
-      recipe_visibility_public: Public
-      recipe_visibility_public_hint: Anyone can find and install.
-      recipe_visibility_private: Private
-      recipe_visibility_private_hint: Breaks external installs.
-      recipe_visibility_unlisted: Unlisted
-      recipe_visibility_unlisted_hint: Anyone with the link can install.
+      recipe_visibility_title: Wie möchtest du veröffentlichen?
+      recipe_visibility_edit_title: Einreichung bearbeiten
+      recipe_visibility_public: Öffentlich
+      recipe_visibility_public_hint: Jeder kann es finden und installieren.
```

```suggestion
      recipe_visibility_public_hint: Jeder kann sie finden und installieren.
```
sie = die Erweiterung, I would argue

---

Ah yes, I had Plugin in mind but it is translated as Erweiterung 👍